src/app/core/user/user.service.ts

```diff
@@ -64,7 +64,7 @@ export class UserService {
     private _disco: DiscoService,
     private _userInfo: UserInfoService
   ) {}
-  $userStatusChecked = new Subject()
```

Comment:
When the user arrives to a headless login (AKA Oauth url page) nothing there will ask for any togglz state.  This means that the `$userStatusChecked` is triggered, but the togglz is not listening to it to call the `config.json` endpoint. 

With this change, even if the `getTogglz()` gets called later on the application life, the call to the config.json will be made.